jsx/DynamicDataTable.js

```diff
@@ -96,13 +96,15 @@ class DynamicDataTable extends Component {
         getFormattedCell={this.props.getFormattedCell}
         freezeColumn={this.props.freezeColumn}
         onSort={this.props.onSort}
+        hiddenHeaders={this.props.hiddenHeaders}
```

Comment:
@HenriRabalais didn't we do away with hiddenHeaders?

Reply:
in the new way of reactifying, yes. but @HenriRabalais and @maltheism both decided that conflict resolver is a little too complex to follow the react template. so this module can't use <FilterableDataTable /> component which did away with hiddenHeaders